composer/models/huggingface.py

```diff
@@ -74,6 +95,145 @@ def __init__(self,
 
         self.labels = None  # set in eval_forward() if exists
 
+    @staticmethod
+    def hf_from_composer_checkpoint(
+        checkpoint_path: str,
+        model_instantiation_class: Optional[Union[Type[transformers.PreTrainedModel],
+                                                  Type['_BaseAutoModelClass']]] = None,
+        model_init_kwargs: Optional[dict] = None,
```

Comment:
The naming `model_init_kwargs` strikes me as a bit vague. I think we tend to use `model_config` as the name for the config default overrides. In any case, I suggest naming it something that mentions the config, since that is what it directly changes.

Reply:
:+1: changed to `model_config_kwargs`